docs/extra-build-config.md

```diff
@@ -0,0 +1,183 @@
+# Optional build configuration
+
+There are a set of ways in which a user can influence different paramenters of
+the build. We list here the ones that are closely related to this BSP or
+specific to it. For the rest please check:
+<http://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html>
+
+## Compressed deployed files
+
+1. Overwrite IMAGE_FSTYPES in local.conf
+  * `IMAGE_FSTYPES = "tar.bz2 ext3.xz"`
```

Comment:
Again, this tidy up and fixing the output is still todo. You've got the right fix here though.